shell/common/isolate_configuration.cc

```diff
@@ -37,9 +37,6 @@ class PrecompiledIsolateConfiguration final : public IsolateConfiguration {
 
```

Comment:
This class should be renamed to something like AppSnapshotIsolateConfiguration

Reply:
Done.